File: Assignment_22/program2.c
// Accept Character from user and check whether it is capital or not (A-Z).

///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Accept Character from user and check whether it is capital or not (A-Z).
// Autor       :  Ankita Anil Patil
// Date        :  31/05/2025
///////////////////////////////////////////////////////////////////////////////


/*
Input : F
Output: TRUE(Capital)

Input : d
Output: FALSE(NOt capital)
*/

#include<stdio.h>

#define TRUE 1
#define FALSE 0

typedef int bool;

bool ChkCapital(char ch)
{
    bool bFlag = FALSE;

    if((ch >= 'A') && (ch <= 'Z'))
    {
        bFlag = TRUE;
    }

    return bFlag;

}

int main()
{
    char cValue = '\0';
    bool bRet = FALSE;

    printf("Enter character \n");
    scanf("%c",&cValue);

    bRet = ChkCapital(cValue);
    
    if(bRet == TRUE)
    {
        printf("%c is capital \n",cValue);
    }
    else
    {
        printf("%c is not capital \n",cValue);
    }

    return 0;
}

File: Assignment_58/program2.java
//  Program to check given strings are Anagram or not 
// This java program will read two strings and check whether they 
// are anagram strings or not. 
/*
    Input : slient
    Output : listen 
 */

////////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.java
// Description :  Accept two strings and check whether they are anagram strings or not.
// Autor       :  Ankita Anil Patil
// Date        :  14/09/2025
///////////////////////////////////////////////////////////////////////////////



import java.util.Scanner; 
class AnagramString  
{ 
    public static boolean isAnagrams(String str1,String str2) 
    {
        if(str1.length() != str2.length())
        {
            return false;
        }
        int i = 0;

        char arr1[] = str1.toCharArray();
        char arr2[] = str2.toCharArray();
        
        int Count[] =new int[26];

        for(i = 0; i < arr1.length; i++)
        {
            if(arr1[i] >= 'a' && arr1[i] <= 'z')
            {
                Count['a' + i]++;
            }

            if(arr2[i] >= 'a' && arr2[i] <= 'z')
            {
                Count['a' + i]--;
            }
            
        }

        for(i = 0; i < 26; i++)
        {
            if(Count[i] != 0)
            {
                break;
                return false;
            }
        }

        
    }  
 } 

 class program2
 {
    public static void main(String[] args)  
    { 
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter String 1 : "); 
        String str1 = sc.next(); 

        System.out.println("Enter String 2 : "); 
        String str2 = sc.next(); 
    
        if(AnagramString.isAnagrams(str1,str2)) 
        { 
            System.out.println("Anagram Strings !!"); 
        } 
        else 
        { 
            System.out.println("Strings are not Anagram !!");
    
        }
    }   
}
